docs/experiments/multiple-focus-metavariables.md

```diff
@@ -0,0 +1,39 @@
+---
+slug: multiple-focus-metavariables
+append_help_link: true
+description: "With this rule, Semgrep matches all pieces of code captured by the focus metavariables."
+---
+
+# Using multiple focus metavariables
+
+Use multiple focus metavariables in one Semgrep rule by specifying each under a new `focus-metavariable` item in a YAML list. With this rule, Semgrep matches all pieces of code captured by the focus metavariables.
+
+:::info
+This feature is using `focus-metavariable`, see [`focus-metavariable`](/writing-rules/rule-syntax/#focus-metavariable) documentation for more information.  
+:::
+
+<iframe src="https://semgrep.dev/embed/editor?snippet=493E" border="0" frameBorder="0" width="100%" height="432"></iframe>
+
+There are two methods of using multiple metavariables. Each of them can lead to different results:
+
+- **Recommended**: Including `focus-metavariable` with more metavariables directly under the `patterns` key returns results regardless of whether there is an overlap between these variables:
+    ```yaml
+        patterns:
+          - pattern: foo($X, ..., $Y)
+          - focus-metavariable: 
+            - $X
+            - $Y
+    ```
+    See the following example:
+    TODO ADD IFRAME
```

Comment:
@enncoded moved the article to experiments. Once this feature is out, let's add the Playground iframe here. :-)

Reply:
```suggestion
TODO - substitute placeholder code example below with simple showcase of YAML list syntax:
<iframe src="https://semgrep.dev/embed/editor?snippet=493E" border="0" frameBorder="0" width="100%" height="432"></iframe>
```

Reply:
```suggestion
    <iframe src="https://semgrep.dev/embed/editor?snippet=D602" border="0" frameBorder="0" width="100%" height="432"></iframe>
```